bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/TypeCreator.java

```diff
@@ -231,22 +231,25 @@ public static ObjectType createObjectType(String typeName, Module module, long f
      * Creates a {@link BStreamType} which represents the stream type.
      *
      * @param constraint the type by which this stream is constrained
+     * @param completionType the type which indicates the completion of the stream
      * @return the new stream type
      */
-    public static StreamType createStreamType(Type constraint) {
```

Comment:
Should we deprecate these and add new methods instead?

Reply:
Fixed: 5a6d295